### ISSUE TITLE
Make commands visible :: 🙈 -> 🙉 

### ISSUE THREAD

**Issue by moonbaseDelta** [egregors/vldc-bot#138]
Here's some code to update bot commands without doing anything special: https://yadi.sk/d/ts4onlho33YFmw 

To install do this:
`pip install pyrogram tgcrypto`

What it needs are `api_id` & `api_hash` to be typed into `config.cfg`, one can obtain 'em there: https://core.telegram.org/api/obtaining_api_id 
After that (just for the first time) the console will ask you for your phone num in order to send a veri-code to bounded tg-account.

And the commands should be inside the `STRING_OF_COMMANDS `variable. How will you build that list depends on thyself, my fellow brethren of colored letters on modern screens.

`BotFather` says about formatting msg with commands:
```
Please use this format:

command1 - Description
command2 - Another description
```

**Comment by egregors** [comment]
@moonbaseDelta looks like we just have a new API for this kind of stuff: https://python-telegram-bot.readthedocs.io/en/stable/telegram.bot.html?highlight=set_my_commands#telegram.Bot.set_my_commands

**Comment by egregors** [comment]
closed by a066eab5bcb01376ff610df6639c39e636365fa1
